src/client/Components/Containers/SpotifyContainer.tsx

```diff
@@ -1,40 +1,40 @@
-import React, { useState, useEffect } from "react";
-import { useSelector } from "react-redux";
-import { HashRouter as Router, Route, Switch } from "react-router-dom";
-import { IRootState } from "../../store/Reducers";
-import { SpotifyLogin } from "../Spotify_Components/SpotifyLogin";
-import axios from "axios";
+// import React, { useState, useEffect } from "react";
```

Comment:
Why are we commenting out the imports instead of removing them here?

Reply:
thanks, deleted